Please get [failure logs here](https://dev.azure.com/vcpkg/c1ee48cb-0df2-4ab3-8384-b1df5a79fe53/_apis/build/builds/100121/artifacts?artifactName=failure%20logs%20for%20arm64-osx&api-version=7.1&%24format=zip) of `arm64-osx`:
```
-- Found Threads: TRUE  
-- Found Protobuf: /Users/vcpkg/Data/installed/arm64-osx/lib/libprotobuf.a (found version "3.21.12") 
-- Could NOT find OpenSSL, try to set the path to OpenSSL root folder in the system variable OPENSSL_ROOT_DIR (missing: OPENSSL_CRYPTO_LIBRARY) 
fatal: not a git repository (or any of the parent directories): .git
-- Looking for clock_gettime
-- Looking for clock_gettime - found
-- Configuring done (0.8s)
CMake Error: The following variables are used in this project, but they are set to NOTFOUND.
Please set them or make sure they are set and tested correctly in the CMake files:
OPENSSL_CRYPTO_LIBRARY (ADVANCED)
    linked by target "OBJ_LIB" in directory /Users/vcpkg/Data/b/braft/src/0ee6a6b410-b96e980f2e.clean/src
    linked by target "OBJ_LIB" in directory /Users/vcpkg/Data/b/braft/src/0ee6a6b410-b96e980f2e.clean/src
OPENSSL_SSL_LIBRARY (ADVANCED)
    linked by target "OBJ_LIB" in directory /Users/vcpkg/Data/b/braft/src/0ee6a6b410-b96e980f2e.clean/src
```

--- a/ports/braft/fix-dependency.patch
+++ b/ports/braft/fix-dependency.patch
@@ -1,5 +1,5 @@
 diff --git a/CMakeLists.txt b/CMakeLists.txt
-index 53ddaed..a851c00 100644
+index 53ddaed..3d75dd5 100644
 --- a/CMakeLists.txt
 +++ b/CMakeLists.txt
 @@ -47,8 +47,9 @@ if ((NOT LEVELDB_INCLUDE_PATH) OR (NOT LEVELDB_LIB))
@@ -14,3 +14,18 @@ index 53ddaed..a851c00 100644
  if ((NOT GFLAGS_INCLUDE_PATH) OR (NOT GFLAGS_LIB))
      message(FATAL_ERROR "Fail to find gflags")
  endif()
+@@ -77,13 +78,7 @@ if (NOT PROTOBUF_PROTOC_EXECUTABLE)
+     set (PROTOBUF_PROTOC_EXECUTABLE "${PROTO_LIB_DIR}/../bin/protoc")
+ endif()
+ 
+-if(CMAKE_SYSTEM_NAME STREQUAL "Darwin")
+-    set(OPENSSL_ROOT_DIR
+-        "/usr/local/opt/openssl"    # Homebrew installed OpenSSL
+-        )
+-endif()
+-
+-include(FindOpenSSL)
++find_package(OpenSSL REQUIRED)
+ 
+ include_directories(
+         ${GFLAGS_INCLUDE_PATH}

--- a/ports/braft/vcpkg.json
+++ b/ports/braft/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "braft",
   "version-date": "2021-26-04",
-  "port-version": 3,
+  "port-version": 4,
   "description": "Consensus algorithm library",
   "homepage": "https://github.com/baidu/braft",
   "license": "Apache-2.0",

--- a/ports/brpc/vcpkg.json
+++ b/ports/brpc/vcpkg.json
@@ -1,11 +1,11 @@
 {
   "name": "brpc",
   "version": "1.6.1",
-  "port-version": 1,
+  "port-version": 2,
   "description": "Industrial-grade RPC framework used throughout Baidu, with 1,000,000+ instances and thousands kinds of services, called \"baidu-rpc\" inside Baidu.",
   "homepage": "https://github.com/apache/incubator-brpc",
   "license": "Apache-2.0",
-  "supports": "!windows & !arm",
+  "supports": "!windows & !(arm & android)",
   "dependencies": [
     "gflags",
     "glog",

--- a/versions/b-/braft.json
+++ b/versions/b-/braft.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "b376168dcf6ec1336dd980ae419f8dcd626bc41b",
+      "version-date": "2021-26-04",
+      "port-version": 4
+    },
+    {
       "git-tree": "af9ff158d1a8f1284b9cc78d4ec816b0a37c7a3d",
       "version-date": "2021-26-04",
       "port-version": 3

--- a/versions/b-/brpc.json
+++ b/versions/b-/brpc.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "3fd0bfb30a96852a7a92e0526c411ef9f95bed83",
+      "version": "1.6.1",
+      "port-version": 2
+    },
+    {
       "git-tree": "5460a635b3cfa64a9857a378a798ded7f60abc74",
       "version": "1.6.1",
       "port-version": 1

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -1334,7 +1334,7 @@
     },
     "braft": {
       "baseline": "2021-26-04",
-      "port-version": 3
+      "port-version": 4
     },
     "breakpad": {
       "baseline": "2023-01-27",
@@ -1350,7 +1350,7 @@
     },
     "brpc": {
       "baseline": "1.6.1",
-      "port-version": 1
+      "port-version": 2
     },
     "brunocodutra-metal": {
       "baseline": "2.1.4",

